README.md

```diff
@@ -69,6 +276,37 @@ that together help the individual
 achieve _all_ of their personal goals
 through a systematic workflow.
 
+
+
+## 1. _Individual_ Person Experience -> [_Single Player_](https://en.wikipedia.org/wiki/Single-player_video_game)
```

Comment:
I'm busy expanding on the **_Individual_ Person's Experience** as the _first_ thing we are building.
related to: https://github.com/dwyl/app/issues/213 (Create Personas)

Reply:
We can add personas in the next PR.